tips/tests/test-rule.py

```diff
@@ -0,0 +1,35 @@
+from objectpath import Tree
+from unittest import TestCase
+import json
+
+def test_rule():
+    inputData = {
+        "info": "2020-09-30",
+        "kinderen": [
+            {
+                "geboortedatum": "1995-01-01"
+            },
+            {
+                "geboortedatum": "2005-01-01"
+            }
+        ]
+    }
+    rule = 'dateTime($.kinderen.geboortedatum[1]) + timeDelta(18, 0, 0, 0, 0, 0) > dateTime($.info)'
+    treedata = Tree(inputData)
+    print("WAT IS DIT", treedata.execute(rule))
+    som = 'dateTime($.kinderen.geboortedatum[1]) + timeDelta(18, 0, 0, 0, 0, 0)'
+    print(treedata.execute(som))
+
+    jsonData = json.dumps(inputData["kinderen"])
+    print("HSHSH", jsonData)
+    geboorte_datum_kinderen = dict(jsonData)
+    print("JSJDJ", geboorte_datum_kinderen_list)
+
+    return inputData
```

Comment:
Is dit een test? dan moet het in een class staan die erft van TestCase. Dan pas word het opgepikt wanneer de tests suite gedraaid word

Reply:
Verder doet dit geen asserts, dus deze test slaagt altijd

Reply:
Dit is de fiddle file. Moet verwijderd worden als het werk klaar is...